conda-recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
   - pip
   run:
   - python >=3.6
-  - bluesky >=1.6.5,<1.11
+  - bluesky >=1.6.5
```

Comment:
Does this need to be re-pinned higher? Or maybe this is for the follow-up?

Reply:
I'll leave this for the followup.  Older versions of bluesky would still work for numpy<2.0, technically.